### PR TITLE
Fix: Not to error when unknown command is passed

### LINKED ISSUE #869: Passing an unknown command to `sonarwhal` breaks it
Running, for example:

```bash
sonarwhal --inti
```

Results in:

```
Unhandled rejection promise:
    uri: undefined
    message: Invalid option '--inti' - perhaps you meant '--init'?
    stack:
Error: Invalid option '--inti' - perhaps you meant '--init'?
    at getOption (/Users/alrra/projects/tmp/node_modules/sonarwhal/node_modules/optionator/lib/index.js:169:15)
    at Object.parse (/Users/alrra/projects/tmp/node_modules/sonarwhal/node_modules/optionator/lib/index.js:398:23)
    at Object.exports.execute (/Users/alrra/projects/tmp/node_modules/sonarwhal/dist/src/lib/cli.js:26:46)
    at run (/Users/alrra/projects/tmp/node_modules/sonarwhal/dist/src/bin/sonarwhal.js:25:34)
    at Object.<anonymous> (/Users/alrra/projects/tmp/node_modules/sonarwhal/dist/src/bin/sonarwhal.js:31:1)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

### PR DESCRIPTION
**Pull request by qzhou1607**
## Pull request checklist

Make sure you:

- [X] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [X] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [X] Added/Updated related tests.

## Short description of the change(s)
Fix #869

### CODE REVIEW

File: packages/sonarwhal/src/lib/cli.ts [hunk 1]
```diff
@@ -67,7 +67,16 @@ const notifyIfNeeded = () => {
 
 /** Executes the CLI based on an array of arguments that is passed in. */
 export const execute = async (args: string | Array<string> | Object): Promise<number> => {
-    const currentOptions: CLIOptions = options.parse(args);
+    let currentOptions: CLIOptions;
+
+    try {
+        currentOptions = options.parse(args);
+    } catch (e) {
+        logger.error(e.message);
```
Comment (sarvaje) [inline packages/sonarwhal/src/lib/cli.ts]: I think we should show the help message.

Comment (sarvaje) [reply packages/sonarwhal/src/lib/cli.ts]: Does `e.message` contain the help message?

If not, I think you should show it.

Comment (qzhou1607) [reply packages/sonarwhal/src/lib/cli.ts]: @sarvaje Yes `e.message` contains the help message.

Comment (sarvaje) [reply packages/sonarwhal/src/lib/cli.ts]: ok, perfect!

**Pull request closed without merging**